### ISSUE TITLE
Adjusting Learning Rate with Increased Batch Size

### ISSUE THREAD

**Issue by Mehrdad-Noori** [lllyasviel/ControlNet#527]
Hello,

I've been using your model with the default batch size of 4 and a learning rate of 1e-5. I'm considering increasing the batch size and possibly using gradient accumulation. Is there a recommended factor by which I should adjust the learning rate when doing so? And any pointers on using gradient accumulation effectively with your model would be appreciated.

Best

**Comment by engrmusawarali** [comment]
You can keep the learning rate same, for any batch size
